compact/hcal/lfhcal_wInsert.xml

```diff
@@ -0,0 +1,179 @@
+<lccdd>
+
+  <info name="lfhcal.xml"
+        title="ECCE LFHCAL with insert cutout"
+        author="@FriederikeBock"
+        url="https://github.com/FriederikeBock"
+        status="development"
+        version="1.0"
+  ><comment/></info>
```

Comment:
We should avoid having two parallel implementations that only differ in the presence (or not) of the insert. It is likely to lead to headaches as parameters are changed in one and not the other.

Reply:
the problem is the positioning of the modules can be different and not easily calculable. That's why having them separate makes sense.